plugins/tours/tests/phpunit/unit/_renderCustomColumnContent.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Tests for _render_custom_column_content().
+ *
+ * @package     spiralWebDb\CornerstoneTours\Tests\Unit
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Unit;
+
+use Mockery as m;
+use Brain\Monkey;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+use function spiralWebDb\CornerstoneTours\_render_custom_column_content;
+
+/**
+ * Class Tests_RenderCustomColumnContent
+ *
+ * @package spiralWebDb\CornerstoneTours\Tests\Unit
+ * @group   tours
+ * @group   admin
+ */
+class Tests_RenderCustomColumnContent extends Test_Case {
+
+	/**
+	 * Instance of the post object for each test.
+	 *
+	 * @var Mockery
+	 */
+	protected $post;
+
+	/**
+	 * The post ID
+	 *
+	 * @var integer
+	 */
+	protected $ID;
+
+	/**
+	 * The post type
+	 *
+	 * @var string
+	 */
+	protected $post_type;
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once TOURS_ROOT_DIR . '/src/admin/wp-list-table.php';
+
+		$this->post = m::mock( 'post' );
+		$this->post->ID = (int) 99;
+		$this->post->post_type = 'tours';
+	}
+
+	/**
+	 *  Test _render_custom_column_content() should echo 'tour_id' when evaluating $column_name.
+	 */
+	public function test_should_echo_tour_id_when_evaluating_column_name() {
+		Monkey\Functions\expect( '_render_custom_column_content' )
+			->once()
+			->with( 'column_name', 'tour_id' )
+			->andReturn( 99 );
+		$expected = $this->post->ID;
+
+		$this->assertSame( $expected, _render_custom_column_content( $column_name, $this->post->ID ) );
+	}
+}
```

Comment:
There are 2 other scenarios that need tests too:

- `public function testShouldEchoTourYearWhenColumnNameIsTourYear()`
- `public function testShouldEchoMenuOrderWhenColumnNameIsMenuOrder()`

```php
public function testShouldEchoTourYearWhenColumnNameIsTourYear() {
	$tour_id = 99;
	$tour_year = 2020;

	Monkey\Functions\expect( 'get_post_meta' )
		->once()
		->with( $tour_id, 'tour_year', true )
		->andReturn( $tour_year );
	Monkey\Functions\expect( 'get_post_field' )->with( 'menu_order', $tour_id )->never();

	$this->expectOutputString( $tour_year );
	_render_custom_column_content( 'tour_year', $tour_id );
}

public function testShouldEchoMenuOrderWhenColumnNameIsMenuOrder() {
	$tour_id = 99;
	$menu_order = 5;

	Monkey\Functions\expect( 'get_post_field' )->with( $tour_id, 'tour_year', true )->never();
	Monkey\Functions\expect( 'get_post_meta' )
		->once()
		->with( 'menu_order', $tour_id )
		->andReturn( $menu_order );

	$this->expectOutputString( $menu_order );
	_render_custom_column_content( 'menu_order', $tour_id );
}
```

Reply:
@hellofromtonya Updated per above.